cfgov/jinja2/v1/youth_employment_success/goals.html

```diff
@@ -0,0 +1,91 @@
+{% from "goals/goal.html" import renderTextarea with context %}
+{% from 'atoms/radio-button.html' import render with context %}
+{% set labels = ['3 to 6 months', '6 to 9 months', '9 months to 1 year', '1 year or more'] %}
+
+<h1 class="h1">Planning how you’ll get to work</h1>
+<p class="lead-paragraph u-mt0">Let’s make a plan for how you’ll get to work—getting to work on time is important for keeping your job. Having a plan and a backup option can help you be prepared so you don’t get in a bind. This tool will help you figure out your best option for getting to work and how to budget for it.</p>
+<p>Meet Mya and Frankie, they’ll help you think about your own long-term goal.</p>
+<div class="content-l block">  
+  <div class="content-l_col-1-2">
+    <img width=400 height=300>
+    <p class="h2 u-mt15">Mya plans to bike to work</p>
+    <p class="u-mt20">Mya decided to bike to work because there’s a bikeshare near her school. Bikeshares allow you to borrow a bike from different self-service bike stations around the city. Mya heads straight to work from class—some days she has time to walk, but usually she’s in a rush and biking is faster.</p>
+    <p class="h3 u-bold">Mya’s long-term goal</p>
```

Comment:
Can you use `<b>Mya’s long-term goal</b>` and ditch the `u-bold` class?

Reply:
Yes I'll do that. I was mistaken about the semantic use of `b` WRT screen readers. I think think those only emphasize content in `strong` tags